src/style.css

```diff
@@ -1,3 +1,82 @@
+:root {
+  --blue: hsl(193, 100%, 43%);
+  --black: hsl(0, 0%, 0%);
+  --white: hsl(0, 0%, 100%);
```

Comment:
When naming variables, you should use the name according to the purpose, and not according to the color, because the color may change to another during the development process. Then you have to update it throughout the code, not just where this variable is created.